tests/testthat/_snaps/derive_blfl.md

```diff
@@ -80,33 +80,33 @@
 
 # derive_blfl sdmt_in validations work
 
-    Required variable `DOMAIN` is missing
+    Required variable `DOMAIN` is missing in `sdtm_in`
```

Comment:
@galachad 
I have removed this file in PR #75 . Can you please review that and then we can look into this PR.

Reply:
@rammprasad the snapshot works as expected, I do not see the reason for removing it.

The issue came from admiraldev and the changes in the latest 1.1.0

Reply:
@rammprasad @galachad : I just revised the PR, and I agree with Adam's changes.